tech_assessment.ipynb

```diff
@@ -45,7 +68,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Read Data"
+    "### Read Data\n",
+    "\n",
+    "Create dataframes for each of the four data files in the `data/` directory."
```

Comment:
Leave out "four" - again - just to see if they follow instructions / work through it